Assignment1/src/Parser.java

```diff
@@ -7,6 +7,8 @@
  * Program purpose: This program will interpret the professor's Z+- code
  */
 
+//Test adding line
```

Comment:
This is a test comment.
```suggestion
//Test adding line suggestion!
```

Reply:
Cool